packages/components/src/Button/button.stories.tsx

```diff
@@ -189,8 +189,8 @@ const renderSize = (
                 {colors.map((color) => (
                   <td key={color} className={styles.color}>
                     {/* To pass the "state" prop (only used for demonstration in storybook),
-                    we must use Clickable instead of Button */}
-                    <Clickable
+                    we must use ClickableStyleWrapper instead of Button */}
+                    <ClickableStyleWrapper
```

Comment:
> Should we rename this component

definite yes from me -- I think you pointed out the confusion of the current name well

> and, if so, to what?

I'm ok with `ClickableStyleWrapper` because it's at least more descriptive. Other thoughts:
- `ClickableStyle` without the "Wrapper"
- `WithClickableStyle` to be more HOC-like

Reply:
ClickableStyle is nice.

Can't think of any other good names. The naming challenge may be a sign that it's a weird abstraction. But of course it's useful to have for the react router links.